developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -10,25 +10,55 @@ Foreman will run with the following requirements (aside from rubygem dependencie
 * Ruby 2.7.x
 * NodeJS 14
 * NPM 6.x
-* PostgreSQL 13
+* PostgreSQL 10+ (12+ recommended)
 
-It’s recommended to install https://github.com/rbenv/rbenv[rbenv] or https://github.com/rvm/rvm[rvm] to manage ruby, and https://github.com/nvm-sh/nvm[nvm] to manage node and npm on your system.
-* For Fedora users, you can use this https://developer.fedoraproject.org/start/sw/web-app/rails.html[guide] for installing: rbenv, Ruby & Bundler.
-(There is no need to install Rails, as it is installed by Bundler when running `bundle install` from Foreman directory).
+https://github.com/rbenv/rbenv[rbenv] or https://github.com/rvm/rvm[RVM] can be used to install compatible Ruby versions. Similarly, https://github.com/nvm-sh/nvm[nvm] can be used to manage node and npm on your system.
 
-Before installing PostgreSQL, check first that you don't already have it installed (by running `rpm -q postgresql`).
+=== Fedora
 
-You will want to make sure that you have postgresql-devel installed so the database gems can install properly.
-Also, you would also need gcc, ruby-devel, libxml-devel, libxslt-devel and libvirt-devel packages.
+Fedora ships with a too new Ruby. It's recommended to install https://github.com/rbenv/rbenv[rbenv] to install a compatible version:
+
+[source, bash]
+....
+sudo dnf install rbenv ruby-build-rbenv nodejs
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+source ~/.bashrc
+rbenv install 2.7.6
```

Comment:
After running this command it asks if you want to make it the default by running `rbenv global 2.7.6`. Is that a good idea to include?

Reply:
I don't think so. We use Ruby 2.7 in Foreman now, but really want to avoid it and move to a more modern version. That's why I chose the `rbenv local` approach: that only changes it for the current directory, and subdirectories.